src/test/java/com/thebois/models/WorldTests.java

```diff
@@ -0,0 +1,38 @@
+package com.thebois.models;
+
+import org.junit.jupiter.api.Test;
+
+import com.thebois.models.tiles.ITile;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class WorldTests {
```

Comment:
Did you check to make sure your coverage is as close to 100% as possible?

Reply:
Not testing the find method now that you mention it, will fix that. And there are no tests for having a world with structures, because there are no structures implmented at the moment. 

Also should we test views? 
Views are not smart so maybe something similar to automated selenium for browsers but for our UI? Might be overkill for this project at the moment.

Reply:
Hmm, if we're writing the tests ourselves then I don't see any clean way to test that a view is actually rendering what it should, and yeah as you say they should be dumb anyways. 

If there is a LibGDX/Java-alternative to Selenium then that would be nice, that can be something we look into next week maybe!